src/queries/text.ts

```diff
@@ -86,8 +88,12 @@ const queryAllByText = (
       return [];
     }
 
-    const results = baseInstance.findAll((node) =>
-      getNodeByText(node, text, options)
+    const results = findAll(
+      baseInstance,
+      (node) => getNodeByText(node, text, options),
+      {
+        hidden: options?.hidden,
+      }
     );
```

Comment:
Can we just foward options?
```suggestion
    const results = findAll(
      baseInstance,
      (node) => getNodeByText(node, text, options),
      options,
    );
```